modules/openapi-generator-gradle-plugin/src/test/kotlin/GenerateTaskDslTest.kt

```diff
@@ -29,6 +29,66 @@ class GenerateTaskDslTest : TestBase() {
         }
     """.trimIndent()
 
+    @Test
+    fun `openApiGenerate should create an expected file structure from URL config`() {
+        val specUrl = "https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master" +
+                "/modules/openapi-generator-gradle-plugin/src/test/resources/specs/petstore-v3.0.yaml"
```

Comment:
I think it's okay for this URL to be all on one line.

```suggestion
        val specUrl = "https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/modules/openapi-generator-gradle-plugin/src/test/resources/specs/petstore-v3.0.yaml"
```